### PR TITLE
Feat/external shap values

### PR DESCRIPTION
**Pull request by linus-mt**

### CODE REVIEW

File: sotai/external.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from .api import post_external_inference
+
+
+def shap(
+    inference_data: pd.DataFrame,
+    shapley_values: np.ndarray,
+    base_values: np.ndarray,
+    name: str,
+):
+    """Uploads a set of shapley values, base values, and inference data to the SOTAI API"""
```
Comment (willbakst) [inline sotai/external.py:14]: Add argument docstring here and include a period at the end of the opening sentence.

File: sotai/external.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import pandas as pd
+from .api import post_external_inference
+
+
+def shap(
+    inference_data: pd.DataFrame,
+    shapley_values: np.ndarray,
+    base_values: np.ndarray,
+    name: str,
+):
```
Comment (willbakst) [inline sotai/external.py:13]: annotate return type

Comment (linus-mt) [reply sotai/external.py]: Dnone.

File: sotai/external.py (hunk 1 above)
Comment (linus-mt) [reply sotai/external.py]: Done.

**Review by willbakst — APPROVED** [review]

**Pull request merged**